src/sql/parts/notebook/cellViews/code.component.ts

```diff
@@ -36,11 +42,22 @@ export const CODE_SELECTOR: string = 'code-component';
 })
 export class CodeComponent extends AngularDisposable implements OnInit {
 	@ViewChild('toolbar', { read: ElementRef }) private toolbarElement: ElementRef;
+	@ViewChild('moreactions', { read: ElementRef }) private moreactionsElement: ElementRef;
 	@ViewChild('editor', { read: ElementRef }) private codeElement: ElementRef;
 	@Input() cellModel: ICellModel;
+	@Input() set model(value: NotebookModel) {
```

Comment:
Minor: I know we put `@Input`s at the top normally, but for get/set we usually place below the constructor.